From 6e81af254fa70cd11c6bffd0343da5eb855954f3 Mon Sep 17 00:00:00 2001
From: Leah Rowe <leah@libreboot.org>
Date: Sun, 5 Nov 2023 11:41:41 +0000
Subject: [PATCH 09/56] dell/e6430: use ME Soft Temporary Disable

i overlooked this. it's set on other boards.

we use me_cleaner anyway, and we set the hap bit, but
this additional setting takes effect even if the ME
region is unaltered. it's just another layer of
disablement, to absolutely ensure Intel ME is not alive

Signed-off-by: Leah Rowe <leah@libreboot.org>
---
 src/mainboard/dell/snb_ivb_latitude/cmos.default | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mainboard/dell/snb_ivb_latitude/cmos.default b/src/mainboard/dell/snb_ivb_latitude/cmos.default
index c883ec40de..088da5d844 100644
--- a/src/mainboard/dell/snb_ivb_latitude/cmos.default
+++ b/src/mainboard/dell/snb_ivb_latitude/cmos.default
@@ -6,5 +6,5 @@ bluetooth=Enable
 wwan=Enable
 wlan=Enable
 sata_mode=AHCI
-me_state=Normal
+me_state=Disabled
 hyper_threading=Enable
-- 
2.47.3

